gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-visibility/api-documentation-v4-visibility.component.html

```diff
@@ -16,7 +16,7 @@
 
 -->
 <div class="visibility">Visibility</div>
-<div class="visibility__subtitle">This can be changed later</div>
+<div *ngIf="label" class="visibility__subtitle">{{ label }}</div>
```

Comment:
Suggestion: change `label` input to a boolean `showVisibilitySubtitle` to make keep the text in this component ?